src/atoms/Icon/index.js

```diff
@@ -1,54 +1,84 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import SVGInline from 'react-svg-inline';
+import SVG from 'react-inlinesvg';
 import classnames from 'classnames';
+import merge from 'lodash/merge';
+import { icons as ICONS } from './constants';
 
-import * as icons from 'assets/images';
 import styles from './icons.module.scss';
 
 function Icon( props ) {
-
-  const icon = icons[props.icon];
-
-  if (!icon) { return null; }
-
   const {
     className,
     onClick,
     height,
     width,
-    inline
+    inline,
+    icon,
+    renderSVGDOM,
+    ...rest
   } = props;
 
+  if (!icon) { return null; }
+
+  const wrapperProps = {
+    className: classnames(
+      styles['icon-wrapper'],
+      onClick && styles.clickable,
+      inline && styles.inline,
+      inline && styles[inline],
+      className
+    ),
+    onClick,
+    style: {
+      width,
+      height
+    },
+    ...rest
+  };
+
+  const sanitizedIcon = icon.replace(/[^a-zA-Z0-9_-]/, '');
+
+  const src = `https://storage.googleapis.com/pg-static-assets/svgs/${sanitizedIcon}.svg`;
```

Comment:
Would there be any benefit to using imgix here potentially?

Reply:
Yes -- potentially. But I don't think it's a big enough win.